scripts/gen_dynamic_openapi.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -e
+
+# Generate an OpenAPI document for all backends.
+#
+# Assumptions:
+#
+#   1. Vault has been checked out at an appropriate version and built
+#   2. vault executable is in your path
+#   3. Vault isn't already running
+
+echo "Starting Vault..."
+if pgrep -x "vault" > /dev/null
+then
+    echo "Vault is already running. Aborting."
+    exit 1
+fi
+
+vault server -dev -dev-root-token-id=root &
+sleep 2
+VAULT_PID=$!
+
+echo "Mounting all builtin backends..."
+
+# Read auth backends
+flag=false
```

Comment:
Nit: the variable names in this file could be improved a bit for clarity:
```
flag -> codeLinesStarted
p -> line
re -> regex
var1 -> plugin
```

Reply:
yes definitely!